src/app/_models/common.ts

```diff
@@ -81,6 +81,7 @@ export interface NavItem {
 export type InfoBarType =
   | 'CASE'
   | 'EVENT'
+  | 'EVENT_P'
```

Comment:
EVENT_P stands for event participant?
it's not clear from the name

Reply:
fixed and pushed